src/components/SubmitIndicator.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { LoadingIndicator } from './LoadingIndicator'
+
+type SubmitTypes = {
+  message: string
+  step: string
+  totalSteps: string
```

Comment:
My only suggestion would be to use numbers instead of strings for the steps and totalSteps, or was this intentional?

```suggestion
  step: number
  totalSteps: number
```

Reply:
It can be changed, I had chosen strings because there was no computation being done to determine the steps/step order/maximum steps.

Reply:
Makes sense. Let's keep it as is.